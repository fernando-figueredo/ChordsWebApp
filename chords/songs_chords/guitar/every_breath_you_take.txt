g g g g g g g g g g g g g em em g c g c g d d d g g g g g