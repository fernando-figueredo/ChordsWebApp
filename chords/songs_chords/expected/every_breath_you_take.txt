g g g g g g g g em em em em em em em em c c c c d d d d g g g g